app/components/Statistics/index.tsx

```diff
@@ -0,0 +1,322 @@
+            name: 'Fylte plasser',
+            count: selectedDataPoint.registrationCount,
+          },
+        ],
+        paidDistribution: [
+          {
+            name: 'Betalt arrangement',
+            count: selectedDataPoint.paidCount,
+          },
+          {
+            name: 'Gratis-arrangement',
+            count: selectedDataPoint.count - selectedDataPoint.paidCount,
+          },
+        ],
+      };
+    };
+
+    if (eventTypeDistribution.length === 0) return;
+
+    setEventTypeStatistics(selectEventTypeDataPoints(selectedEventType.value));
+  }, [eventTypeDistribution.length, selectedEventType]);
+
+  if (eventTypeDistribution.length === 0 || !eventTypeStatistics) {
+    return (
+      <Content>
+        <Helmet title="Statistikk" />
+        <h1>Statistikk</h1>
+        <LoadingIndicator loading={true} />
+      </Content>
+    );
+  }
+
+  return (
+    <Content>
+      <Helmet title="Statistikk" />
+      <h1>Statistikk</h1>
+      <Flex wrap gap="0.4rem" className={styles.selectPeriodContainer}>
+        <Select
+          name="select-year"
+          value={
+            PAST_FIVE_YEARS_OPTIONS.find(
+              (pastYearOption) => pastYearOption.value === Number(year)
+            ) ?? PAST_FIVE_YEARS_OPTIONS[0]
+          }
+          options={PAST_FIVE_YEARS_OPTIONS}
+          onChange={(selectedYear) => {
+            props.onSelect(String(selectedYear.value), String(semester));
+          }}
+          isClearable={false}
+          theme={selectTheme}
+          styles={selectStyles}
+          className={styles.selectPeriod}
+        />
+
+        <Select
+          name="select-semester"
+          value={
+            SEMESTER_OPTIONS.find(
+              (semesterOption) => semesterOption.value === semester
+            ) ?? SEMESTER_OPTIONS[0]
+          }
+          options={SEMESTER_OPTIONS}
+          onChange={(selectedSemester) => {
+            props.onSelect(String(year), selectedSemester.value);
+          }}
+          isClearable={false}
+          theme={selectTheme}
+          styles={selectStyles}
+          className={styles.selectPeriod}
+        />
+      </Flex>
+
+      <PieChartWithLabel
+        label={'Arrangementstyper'}
+        distributionData={eventTypeDistribution.map((eventTypeDataPoint) => ({
+          name: EVENT_CONSTANTS[eventTypeDataPoint.eventType],
+          count: eventTypeDataPoint.count,
+        }))}
+        displayCount={true}
+        namedChartColors={Object.keys(COLOR_CONSTANTS).reduce(
+          (acc, colorConstantKey) => ({
+            ...acc,
+            [EVENT_CONSTANTS[colorConstantKey]]:
+              COLOR_CONSTANTS[colorConstantKey],
+          }),
+          {}
+        )}
+      />
+
+      <h4>Arrangementstyper per uke</h4>
+      <BarChart
+        width={1000}
+        height={400}
+        data={weekNumberDistribution}
+        margin={{
+          top: 10,
+          right: 30,
+          left: 0,
+          bottom: 0,
+        }}
```

Comment:
I'd prefer if you use `ResponsiveContainer` for this, like how it's used here: https://github.com/webkom/lego-webapp/blob/master/app/routes/events/components/EventAttendeeStatistics.tsx#L281